app/views/shared/_new_gear_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= simple_form_for @gear do |f| %>
+  <%= f.input :name %>
+  <%= f.input :category %>
+  <%= f.input :description %>
+  <%= f.input :size %>
+  <%= f.input :address %>
+  <%= f.hidden :user_id, value: @user.id %>
+  <%= f.button :submit %>
```

Comment:
This needs to have a field for attaching a photo via attachionary, will happen on Wednesday.